src/main/java/ru/spbstu/model/Variant.java

```diff
@@ -51,6 +56,7 @@
     @ToString.Exclude
     @EqualsAndHashCode.Exclude
     @OneToMany(mappedBy = "variant", cascade = CascadeType.ALL, orphanRemoval = true)
+    @JsonProperty("annotations")
     private Set<Annotation> annotations = new HashSet<>();
 
     public Variant addAnnotation(final Annotation annotation) {
```

Comment:
Just a note: it is a best practice to keep Entities and DTOs separate, even if they are identical. Not critical for our project, but important in production code.

Reply:
Thank you for your comment! 
We have discussed this issue with @Kirvolque and decided that combining DTO and Entity inside single _Variant_ class is acceptable for study project

Reply:
Yeah it's okay for now.